src/index.ts

```diff
@@ -74,7 +76,13 @@ const releaseTagTemplateRegex = /{release_tag}/g;
     const labels = parseLabels(labelTemplate);
     const skipLabels = parseLabels(skipLabelTemplate);
 
-    const linkedIssuesPrs = new Set<number>();
+    interface PR {
+      number: number;
+      title: string;
+      author: string;
+    }
+
+    const linkedIssuesPrs = new Set<PR>();
```

Comment:
This potentially breaks deduplication, as each pushed object is unique, and the set doesn't ignore duplicates. Maybe trying a `Map<number, { title: string, author: string }>`?

Reply:
I've done a few updates based on our  use case since opening the PR. See here, let me know what you think. Will try to resolve merge conflicts in the next few days

Reply:
See: https://github.com/apexskier/github-release-commenter/pull/487